### PR TITLE
Changing the shell used to run commands in the documentation (from `sh` to `bash`).

### PR DESCRIPTION
**Pull request by Lucaweihs**

### CODE REVIEW

File: projects/objectnav_baselines/experiments/ithor/objectnav_ithor_base.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def __init__(self):
         self.TRAIN_GPU_IDS = [0, 1, 2, 3, 4, 5, 6]
         self.VALID_GPU_IDS = [7]
         self.TEST_GPU_IDS = [7]
-        self.ADVANCE_SCENE_ROLLOUT_PERIOD = 10 ** 13
+        self.ADVANCE_SCENE_ROLLOUT_PERIOD = None
```
Comment (klemenkotar) [inline projects/objectnav_baselines/experiments/ithor/objectnav_ithor_base.py:59]: Why did you change this? (To be honest I just inherited this value from older configs I saw and never touched it, I am just curious as to why we are changing it now)

Comment (jordis-ai2) [reply projects/objectnav_baselines/experiments/ithor/objectnav_ithor_base.py:59]: I think this was inherited from a made-in-Jordi hack :D I much prefer having None...
